src/fides/core/export.py

```diff
@@ -142,11 +144,19 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
     """
 
     custom_columns = {}
-    formatted_data_uses, custom_data_uses = format_data_uses(
-        server_resources["data_use"]
-    )
-    formatted_data_subjects, custom_data_subjects = format_data_subjects(
-        server_resources["data_subject"]
+    (
+        formatted_data_uses,
+        custom_data_use_keys,  # pylint: disable=unused-variable
```

Comment:
this doesn't feel well-documented, I think this is here in the eventuality someone would want to use the below code that is also commented out?

Reply:
scratch this, I'm on a refactor spree